supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesAsyncBackendJobExecutionActor.scala

```diff
@@ -524,47 +524,54 @@ class JesAsyncBackendJobExecutionActor(override val jobDescriptor: BackendJobDes
 
   private def metadataKey(key: String) = MetadataKey(workflowId, Option(metadataJobKey), key)
 
-  /**
-    * Turns a GCS path representing a workflow input into the GCS path where the file would be mirrored to in this workflow:
-    * task x {
-    *  File x
-    *  ...
-    *  Output {
-    *    File mirror = x
-    *  }
-    * }
-    *
-    * This function is more useful in working out the common prefix when the filename is modified somehow
-    * in the workflow (e.g. "-new.txt" is appended)
-    */
-  private def gcsInputToGcsOutput(inputValue: WdlValue): WdlValue = {
-    // Convert to the local path where the file is localized to in the VM:
-    val vmLocalizationPath = gcsPathToLocal(inputValue)
-
-    vmLocalizationPath match {
-      // If it's a file, work out where the file would be delocalized to, otherwise no-op:
-      case x : WdlFile =>
-        val delocalizationPath = callRootPath.resolve(vmLocalizationPath.valueString).toString
-        WdlFile(delocalizationPath)
-      case a: WdlArray => WdlArray(a.wdlType, a.value map { f => gcsInputToGcsOutput(f) })
-      case m: WdlMap => WdlMap(m.wdlType, m.value map { case (k, v) => gcsInputToGcsOutput(k) -> gcsInputToGcsOutput(v) })
-      case other => other
-    }
-  }
-
-  private def customLookupFunction(alreadyGeneratedOutputs: Map[String, WdlValue]): String => WdlValue = toBeLookedUp => {
+  // TODO: This might have been necessary for some obscure use case but was breaking simple tests and is therefore no longer used:
+//  /**
+//    * Turns a GCS path representing a workflow input into the GCS path where the file would be mirrored to in this workflow:
+//    * task x {
+//    *  File x
+//    *  ...
+//    *  Output {
+//    *    File mirror = x
+//    *  }
+//    * }
+//    *
+//    * This function is more useful in working out the common prefix when the filename is modified somehow
+//    * in the workflow (e.g. "-new.txt" is appended)
+//    */
+//  private def gcsInputToGcsOutput(inputValue: WdlValue): WdlValue = {
+//    // Convert to the local path where the file is localized to in the VM:
+//    val vmLocalizationPath = gcsPathToLocal(inputValue)
+//
+//    vmLocalizationPath match {
+//      // If it's a file, work out where the file would be delocalized to, otherwise no-op:
+//      case x : WdlFile =>
+//        val delocalizationPath = callRootPath.resolve(vmLocalizationPath.valueString).toString
+//        WdlFile(delocalizationPath)
+//      case a: WdlArray => WdlArray(a.wdlType, a.value map { f => gcsInputToGcsOutput(f) })
+//      case m: WdlMap => WdlMap(m.wdlType, m.value map { case (k, v) => gcsInputToGcsOutput(k) -> gcsInputToGcsOutput(v) })
+//      case other => other
+//    }
+//  }
```

Comment:
Would prefer more discussion and then deletion of the commented out code.

_-OR-_

The block should be marked nearby with `PBE` so we come back to it later for cleanup.